pkg/ddevapp/mutagen.go

```diff
@@ -729,11 +729,11 @@ func CheckMutagenVolumeSyncCompatibility(app *DdevApp) (ok bool, volumeExists bo
 	if mutagenSyncExists {
 		mutagenLabel, mutagenSyncLabelErr = GetMutagenSyncLabel(app)
 		if mutagenSyncLabelErr != nil {
-			util.Warning("mutagen sync %s exists but unable to get label: %v", app.Name, mutagenSyncLabelErr)
+			util.Warning("mutagen sync session '%s' exists but unable to get sync label '%s': '%v' This is normal on upgrade from v1.21.6; error=%v", app.Name, mutagenSignatureLabelName, mutagenLabel, mutagenSyncLabelErr)
```

Comment:
Why do we need `mutagenSignatureLabelName` here? It's anyway a constant.

Reply:
The only reason is to make debugging it easier. I'm concerned about the various mutagen failures we're hearing about.

Reply:
But does this constant change from time to time? It's a constant, so normally not interesting seeing it in a debug output. But anyway not a blocker, just don't get it why this could be useful.

Reply:
It doesn't change, but you'd have to look at the code to figure out what was missing to use the proper mutagen command to see the labels. It's just about making it easy to debug, should we need to do so.